packages/transaction-controller/src/types.ts

```diff
@@ -100,6 +102,38 @@ export interface Transaction {
   estimateGasError?: string;
 }
 
+/**
+ * @type TransactionReceipt
+ *
+ * TransactionReceipt representation
+ * @property effectiveGasPrice - Effective gas price the transaction was charged at
+ * @property gasUsed - Gas used in the transaction
+ * @property l1Fee - Total used gas in hex
+ * @property status - The status of the transaction
+ */
+export interface TransactionReceipt {
```

Comment:
Since we're adding a new interface, should we instead put the doc inside it on each property as done with the `RemoteTransactionSourceRequest` interface for example? It seems cleaner having the description next to each property.

Reply:
Fixed in https://github.com/MetaMask/core/pull/1592/commits/c84227169c0c8b168a3171b3048b4939a1d07411